index.html

```diff
@@ -102,51 +102,97 @@ <h1 id="title">Dubbing and Audio description Profiles of TTML2</h1>
-          <li>Notating dialogue with character information and other annotations</li>
-          <li>Generating localization notes to guide further adaptation</li>
-          <li>Translating the dialogue to a target language</li>
-          <li>Adapting the translation to the dubbing and subtitling specifications; ex. matching the actor’s lip movements in the case of dubs and considering reading speeds and shot changes.</li>
-        </ul>
-      </p>
-      <p>The result of creating a <a>dubbing script</a> can be useful as a starting point for creation of subtitles or closed captions in alternate languages.
-        This specification is designed to facilitate the addition of, and conversion to,
-        subtitle and caption documents in other profiles of TTML, such as [[ttml-imsc1.2]],
-        for example by permitting subtitle styling syntax to be carried in DAPT documents.
-        Alternatively, styling can be applied to assist voice artists when recording scripted dialogue or <a>audio descriptions</a>.
-      </p>
-      <p>Creating audio description content is also a complex process with similar steps.
-        An <dfn>audio description</dfn>, also known as video description, is an audio service
-        to assist viewers who can not fully see a visual presentation to understand the content.
-        It is the result of the audio rendition of one or more <a>descriptions</a>
-        mixed with the audio associated with the programme prior to any mixing with audio description
-        (sometimes referred to as <dfn>main programme audio</dfn>),
-        at moments when this does not clash with dialogue, to deliver an <dfn>audio description mixed audio track</dfn>.
-        A <dfn>description</dfn> is a set of words that describe an aspect of the programme presentation,
-        suitable for rendering into audio by means of vocalisation and recording
-        or used as a text alternative source for text to speech translation, as defined in [[WCAG20]].
-        More information about what <a>audio description</a> is and how it works can be found at [[WHP051]].
-      </p>
-
-      <p class="issue" data-number="28"></p>
-      <p>This specification defines DAPT, a TTML-based format for the exchange of timed text content among authoring and prompting tools in the localization and <a>audio description</a> pipeline.
-        A DAPT file is designed to carry pertinent information for dubbing or <a>audio description</a> such as type of script, dialogue, descriptions, timing, metadata, original language text, transcribed text, language information, and to be extensible for future annotations.
-        This specification first defines the data model (see [[[#data-model]]]) for DAPT scripts and then its representation as a TTML document with restrictions (see [[[#ttml-format]]]).</p>
+      <section id="intro-overview">
+        <h3>Transcripts</h3>
```

Comment:
I'm not sure adding the term "transcript" adds value. We already have the term "script". I'd suggest renaming this section: "Scripts" or "Common script needs in localization and audio description pipelines".

Reply:
I need to improve the wording here, because I do think there's value, but I don't think I've explained it adequately in the text.

It's a deliberate choice on my part to use "trancript" because the way I understand it, a transcript is a text version of pre-existing content, i.e. someone watches the video and writes stuff about it down. That contrasts with "script" which is a plan for how to create new content, i.e. someone will read the script and use it to make a new audio recording.

One of my motivations here is that I'm told that there are likely to be other uses for transcript documents that could reasonably be DAPT documents, for example in support of accessibility use cases for cognitive impairments.

I'll do an editing pass to try to clarify this, and hopefully that will improve things...

Reply:
See if 6e5675c helps...